### DISCUSSION TITLE
The default function of the right mouse button under Android is the return function, which cannot be used normally like in WIN10 environment.

### DISCUSSION THREAD

**Discussion by myzyvox** [rustdesk/rustdesk#9519] [General]
The default function of the right mouse button under Android is the return function, which cannot be used normally like in WIN10 environment.

**Comment by rustdesk** [comment]
Do you mean android -> windows or windwos -> android? Do you expect `return` or anything else?

**Reply by rustdesk** [reply]
BTW, I found something like this https://www.quora.com/My-mouse-s-right-click-act-as-back-How-do-I-fix-that-I-want-the-right-button-register-right-click-in-Android#:~:text=Right%2Dclick%20is%20a%20shortcut,done%20by%20a%20long%2Dpress.
It seems `return` behavior can be modified on your phone settings.
